dm/syncer/dbconn/db.go

```diff
@@ -210,8 +207,7 @@ func (conn *DBConn) ExecuteSQLWithIgnore(
 					zap.Error(err))
 				return true
 			}
-			// TODO: move it to above IsRetryableError
-			return isRetryableError(err)
+			return false
```

Comment:
Should `ErrColumnInChange` be retried? In https://github.com/pingcap/tidb/issues/35102 , this is supposed to be retried.

Reply:
included in https://github.com/pingcap/tiflow/blob/7974dd31256db1d5cad1199dc36cd4aa79fa7541/dm/syncer/dbconn/db.go#L114

Reply:
please read PR description and title

Reply:
https://github.com/pingcap/tidb/pull/35394/files#diff-dd4803f526cf56d6adca5e33b6a76193a035cd98ed31ed8e50b1698635c296c8